fuel_test/configure_interfaces.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import argparse
+import yaml
+import itertools
+
+
+def configure_interfaces(ifaces, actions):
+    all_networks = [iface["assigned_networks"] for iface in ifaces]
+    all_networks = list(itertools.chain(*all_networks))
+    for iface_name in actions:
```

Comment:
I don't understand what's going on here; you've got "actions" which according to this line is a list of interfaces...
Perhaps this method needs some comments to explain what it's doing?